lib/oauth2-loopback.js

```diff
@@ -828,7 +828,7 @@ module.exports = function(app, options) {
     app.post(options.loginPath || '/login',
       passport.authenticate('loopback-oauth2-local',
       {successReturnToOrRedirect: '/',
-        failureRedirect: options.loginPage || '/login'}));
+        failureRedirect: options.loginPage + '?loginFailed' || '/login'}));
```

Comment:
Do we allow loginPage such as `https://localhost:3000/login?transactionId=12`? Simply appending `?loginFailed` can be problematic.